CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 -   [ENH] `change_index_dtype` added. - @samukweku Issue #1314
 -   [ENH] Add `glue` and `axis` parameters to `collapse_levels`. - Issue #211 @samukweku
 -   [ENH] `row_to_names` now supports multiple rows conversion to columns. - @samukweku Issue #1333
+-   [ENH] `truncate_datetime` now uses a vectorized option. -@samukweku #1337
```

Comment:
@samukweku is this changelog fully accurate? If not, could you help flesh it out please? To get things started, I'll do a quick run of gitbot to get a summary out for you.